src/MoBi.Presentation/Presenter/EditPassiveTransportBuildingBlockPresenter.cs

```diff
@@ -48,6 +45,8 @@ public class EditPassiveTransportBuildingBlockPresenter : EditBuildingBlockPrese
          _viewItemContextMenuFactory = viewItemContextMenuFactory;
          _editTransportBuilderPresenter = editTransportBuilderPresenter;
          _transportBuilderToDTOTransportBuilderMapper = transportBuilderToDTOTransportBuilderMapper;
+         _editTransportBuilderPresenter.RemoveFormulaType<TableFormulaWithOffset>();
```

Comment:
I do not think that TableFormulaWithOffset or SumFormula makes sense for any Transport type
It definitely does not make sense for PassiveTransport.
@Yuri05 If you agree, I would like to move this condition to the TransportBuilderPresenter unless you see a use of `TableFormulaWithOffset` or `SumFormula`...